### PR TITLE
Fix/fix esr certificate semantic

### PR DESCRIPTION
**Pull request by ds-jhartmann**
Enabled deserialization of certifiacte object
Added EsrCertificate to IRS request
Increased polling rate to every 2s 30 times

### CODE REVIEW

File: irs-api/src/main/resources/application.yml [hunk 1]
```diff
@@ -102,8 +102,8 @@ resilience4j:
         baseConfig: default
       waiting-for-completed:
         resultPredicate: 'org.eclipse.tractusx.esr.irs.JobIsRunning'
-        maxAttempts: 10
-        waitDuration: 10s
+        maxAttempts: 30
+        waitDuration: 2s
```
Comment (ds-ext-kmassalski) [inline irs-api/src/main/resources/application.yml:106]: Retry every 2s seems to be to fast :)

**Review by ds-ext-kmassalski — CHANGES_REQUESTED** [review]

File: irs-api/src/main/resources/application.yml (hunk 1 above)
Comment (ds-jhartmann) [reply irs-api/src/main/resources/application.yml:106]: Maybe for the first 10 or 15 seconds, but after the IRS job is finished, we want to get the result as fast as possible so we don't run into the 3 minute gateway timeout of DEV

File: esr/src/main/java/org/eclipse/tractusx/esr/irs/model/shell/Shell.java [hunk 2]
```diff
@@ -52,7 +52,7 @@ public class Shell {
     public Optional<String> findManufacturerIdIfValid() {
         return this.specificAssetIds
                 .stream()
-                .filter(assetId -> "ManufacturerId".equals(assetId.getKey()))
+                .filter(assetId -> "manufacturerId".equals(assetId.getKey()))
```
[the remark this answers is not in the document]
Comment (ds-jhartmann) [reply esr/src/main/java/org/eclipse/tractusx/esr/irs/model/shell/Shell.java:55]: Do you have a documentation for this? I tried to find something in regards to that but didn't so I just copied it from SerialPartTypization and Batch

File: irs-api/src/main/resources/application.yml (hunk 1 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/main/resources/application.yml:106]: Ok

**Review by ds-ext-kmassalski — APPROVED** [review]

**Pull request merged**